easybuild/easyblocks/f/fftw.py

```diff
@@ -179,7 +184,10 @@ def sanity_check_step(self):
         }
 
         extra_files = []
+        cpu_arch = get_cpu_architecture()
         for (prec, letter) in [('double', ''), ('long_double', 'l'), ('quad', 'q'), ('single', 'f')]:
+            if prec == 'quad' and cpu_arch == POWER:
+                continue
```

Comment:
this checking should be done earlier, and `with_quad_prec` should be hard set to `False` instead, i.e. in `run_all_steps` before the `for prec in FFTW_PRECISION_FLAGS:` loop...